doc-content/enterprise-only/openshift/operator-deploy-security-proc.adoc

```diff
@@ -54,17 +58,16 @@ For example:
 admins = kie-server,rest-all,admin
 ----
 +
-To enable the use of this file, make the following changes:
+To enable the use of this string or file, make the following changes:
 +
-.. Under *RoleMapper*, in the *Roles properties file* field, enter the fully qualified path name of the role mapping configuration file, for example, `/opt/eap/standalone/configuration/rolemapping/rolemapping.properties`.
-.. If you want to replace roles defined in the authentication system with roles that you define in the mapping file, select the *Replace roles* box. Otherwise, both the roles defined in RH-SSO or LDAP and the roles defined in the configuration file are available.
-.. In the fields under *RoleMapper Configuration object*, select the *Kind* of the object that provides the file (`ConfigMap` or `Secret`) and enter the *Name* of the object. This object is automatically mounted on {CENTRAL} and {KIE_SERVER} pods in the path that you specified for the role mapping configuration file.
+.. Under *RoleMapper*, in the *Roles properties file* field, enter the role configuration string or the fully qualified path name of the role mapping configuration file, for example, `/opt/eap/standalone/configuration/rolemapping/rolemapping.properties`.
+.. If you want to replace roles defined in the authentication system with roles that you define in the mapping string or file, select the *Replace roles* box. Otherwise, both the roles defined in RH-SSO or LDAP and the roles defined in the configuration file are available.
```

Comment:
replace roles does not exists anymore.
needs to set keep non mapped or keep mapped.

Reply:
It was there in the image you gave me. Could you possibly make a newer image?